src/Fantomas.Core.Tests/TypeAnnotationTests.fs

```diff
@@ -266,25 +262,27 @@ let private asJson (arm: IArmResource) =
        >
 """
 
+let forcedLongDefnConfig =
```

Comment:
I'm not a fan of this configuration.
The only thing you are trying to achieve is to spread things over multiple lines, using `MaxLineLength = 30`. All the other settings are irrelevant and misleading to readers.

Reply:
Yeah, fully agree, this latest commit was mainly a WIP